### PR TITLE
Remove run time config for User Agent

### PR DESCRIPTION
**Pull request by ts-leojoseph-r**
# Description
Remove run time configuration for UA. UA can be configured only at the build time.

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

**Comment by corycaywood** [comment]
What was the reason behind this change? All of the other configuration options can be set via either .plist or at runtime, so this breaks with the expectations of the SDK user.

If we keep this change, we should probably add some note to the README about runtime config changes being not supported for the user agent.

**Comment by ts-leojoseph-r** [comment]
> What was the reason behind this change? All of the other configuration options can be set via either .plist or at runtime, so this breaks with the expectations of the SDK user.
> 
> If we keep this change, we should probably add some note to the README about runtime config changes being not supported for the user agent.

I checked with @ts-nitish-a-jain and I believe we should allow the Host application to customize the User-Agent string only @ build time. That's why I made this change

File: MiniApp/Classes/Environment.swift [hunk 1]
```diff
@@ -54,7 +52,7 @@ internal class Environment {
     }
 
     var hostAppUserAgentInfo: String {
-        return value(for: customHostAppUserAgentInfo, fallback: .hostAppUserAgentInfo)
+        return value(for: .none, fallback: .hostAppUserAgentInfo)
```
Comment (Climbatize) [inline MiniApp/Classes/Environment.swift:55]: you could directly return hostAppUserAgentInfo from plist if you never intend to override it.

**Comment by ts-leojoseph-r** [comment]
@corycaywood @Climbatize , Main reason why we don't want to have them enabled at runtime as well is that there is a chance the User Agent may get added more than once(host app can modify the values anytime) in WebView.

**Comment by corycaywood** [comment]
> @corycaywood @Climbatize , Main reason why we don't want to have them enabled at runtime as well is that there is a chance the User Agent may get added more than once(host app can modify the values anytime) in WebView.

Probably this is fine to not support changing it at runtime. But please update the user guide with a note so that it's clear this value is an exception and can't be changed at runtime.

File: MiniApp/Classes/Environment.swift (hunk 1 above)
Comment (ts-leojoseph-r) [reply MiniApp/Classes/Environment.swift]: Done

**Pull request merged**